app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
+    before_filter :authorize_token
+    before_filter :my_load_and_authorize_resource, only: [:update, :destroy, :show]
+
+    attr_reader :item
+
+    def create
+      head :created, location: sufia.api_item_path(actor.create_file_from_item)
+    end
+
+    def update
```

Comment:
@inukshuk If you check out `spec/controllers/api/items_controller_spec.rb` you'll see three tests that make sure a 404 is returned when if a specified ID is not found on `show`, `update`, and `destroy`.